test/requests/path_completion_test.rb

```diff
@@ -27,7 +27,7 @@ def test_completion_command
     }
     end_position = {
       line: 0,
-      character: document.source.rindex('"'),
+      character: T.must(document.source.rindex('"')) - 1,
```

Comment:
These off by one errors need to be fixed in YARP. I wonder how hard it would be to help them get this done, so that we can stop altering our tests.